projects/dotnet-example.yml

```diff
@@ -104,3 +104,12 @@ results:
       - noauth # Command dropDatabase requires authentication
       - plain # Unsupported authentication mechanism "PLAIN"
       - sha1 # Unsupported authentication mechanism "SCRAM-SHA-1"
+
```

Comment:
driver  and mongo tools configs have `ferretdb2-dev-branch` config, `ycsb` configs have `ferretdb2-branch`. But I didn't make change to `mongo-core-test`, it's already too slow and adding another configuration timed out.